resources/public/api/conf/1.0/schemas/list_charges_response.json

```diff
@@ -0,0 +1,124 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "title": "List self assessment charges",
+  "description": "List self assessment charges",
+  "type": "object",
+  "properties": {
+    "charges": {
+      "description": "A list of charges.",
+      "type": "array",
+      "items": {
+        "title": "charge",
+        "type": "object",
+        "properties": {
+          "taxYear": {
+            "description": "The tax year to which the charge relates.",
+            "type": "string",
+            "pattern": "^\\d{4}-\\d{2}$",
+            "example": "2019-20"
+          },
+          "id": {
+            "description": "A unique identifier for the charge.",
+            "type": "string",
+            "pattern": "^[0-9A-Za-z]{1,12}$"
+          },
+          "transactionDate": {
+            "description": "The date the charge was created.",
+            "type": "string",
+            "pattern": "^\\d{4}-\\d{2}-\\d{2}$"
+          },
+          "type": {
+            "description": "A description of the type of charge.",
+            "type": "string",
+            "pattern": "^[0-9a-zA-Z\\s]{1,30}$"
```

Comment:
I'm not sure if this needs a regex.

Reply:
There's one in the spec, I'm happy to just put a max length on it or something. Though since this a response body not a request body that's probably not necessary... your call

Reply:
My only gripe with it is that we don't include that regex for the same field in charge history. Adding a max length instead (and also for the same field in charge history) would be a decent compromise I think.

Reply:
I added a max length to the schema. It doesn't display in the description, it only breaks the example when the field it's applied to go over it. I'll just remove the regex.